boards/tutorials/nrf52840dk-thread-tutorial/src/main.rs

```diff
@@ -132,6 +135,30 @@ pub unsafe fn main() {
     //--------------------------------------------------------------------------
     // NONVOLATILE STORAGE
     //--------------------------------------------------------------------------
+    // Create the software-based SHA engine.
+    let sha = components::sha::ShaSoftware256Component::new()
+        .finalize(components::sha_software_256_component_static!());
+
+    // Create the credential checker.
+    let checking_policy = components::appid::checker_sha::AppCheckerSha256Component::new(sha)
+        .finalize(components::app_checker_sha256_component_static!());
+
```

Comment:
We probably don't want to use this sha256 checker for now just to reduce complexity for running the tutorial. We can just use the null permit all checker.

Reply:
Changed to use the null checker https://github.com/tock/tock/pull/4109/commits/6307c2ce06632fdc60228428f8e040098eae79f8